qiskit/algorithms/eigen_solvers/numpy_eigen_solver.py

```diff
@@ -103,6 +103,10 @@ def filter_criterion(
     def supports_aux_operators(cls) -> bool:
         return True
 
+    @classmethod
+    def supports_transition_amplitudes(cls) -> bool:
```

Comment:
I will note that the similar supports_aux_operators is part the minimum_eigen_solver interface so given any eigensolver instance that method will be there. If at some point we end up with transition amplitudes as part of that interface definition then I can see having a method like this. We can keep this if you like, but at present you will more have to know that transition amplitudes are only supported by this class since this method, and the one implementing the function are only here for now.

Reply:
Indeed, we can set it by default to false int the abstact minimum_eigen_solver.